tools/build/tests/_data/meta/che-theia-meta.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 publisher: eclipse
-name: che-theia
+name: che/theia/next
```

Comment:
if you're only doing `name.replace(/\//gi, '-');` (line 96 of https://github.com/eclipse-che/che-plugin-registry/pull/1444/files#diff-26afe00b049763ef52d61d3be9c67b6858979f7ce164440487906fceb5f9a3a0 ) how will this turn into `theia` instead of `chetheianext` ?

Reply:
If we look at che-editors.yaml https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml#L5
we have name: `eclipse/che-theia/next` (publisher/**name**/version), final generated devfile.yaml will be located in /v3/plugins/eclipse/che-theia/next/ folder and the value of name attribute will be che-theia.